roles/swift-account/tasks/main.yml

```diff
@@ -11,7 +11,6 @@
 - name: account-server config
   template: src=etc/swift/account-server.conf mode=0640
             dest=/etc/swift/account-server.conf owner=swift group=swift
-  notify: restart swift-account-services
 
```

Comment:
which case? The case of config file change but code not changing is certainly not handled by that.